standard_template/standard/clause_specification_text.adoc

```diff
@@ -1117,7 +1118,7 @@ Coverage example:
 ### 10.5. MultiPointSeries
 
 - A domain with MultiPointSeries domain type MUST have the axes `"composite"` and `"t"`.
-- The axis `"composite"` MUST have the data type `"tuple"` and the coordinate identifiers `"x","y","z"` or `"x","y"`.
+- The axis `"composite"` MUST have the data type `"tuple"` and the coordinate identifiers `"x","y","z"` or `"x","y"`, in that order.
```

Comment:
do these tuple orders need to be consistent with the coordinate reference system definition.

the CRSs mandate a coordinate tuple order, some may mandate the inverse of this order
how is that handled?

Reply:
The tuple orders do not have to be consistent with the CRS definition. The mapping is handled by the [Reference System Connection object](https://covjson.org/spec/#612-reference-system-connection-objects). This is a general rule in CoverageJSON, not specific to the domain types.